test_notebooks.py

```diff
@@ -24,12 +24,15 @@
 SI = pint.UnitRegistry()
 
 
+def find_files(path_to_folder_from_project_root=".", file_extension=".ipynb"):
```

Comment:
let's also use it in here: https://github.com/open-atmos/devops_tests/blob/269f8685a1973f8a788562ddbc918e6629017893/test_todos_annotated.py#L26

Reply:
Added